app/address/address-lookup.js

```diff
@@ -12,7 +12,7 @@ function addressLookup(postcode) {
     .then(processBody)
     .catch((e) => {
       if (e.logMessage) {
-        console.error(e.logMessage + ', ErrorId=' + e.error);
+        // console.error(e.logMessage + ', ErrorId=' + e.error);
```

Comment:
Please see other reviews in this PR for how to implement the logging correctly, rather than losing the error logging altogether.

Reply:
DONE